src/api/modelTypes.js

```diff
@@ -350,8 +350,7 @@ export type NarrowElement =
  | {| +operator: 'pm-with', +operand: string | $ReadOnlyArray<UserId> |}
  | {| +operator: 'sender', +operand: string | UserId |}
  | {| +operator: 'group-pm-with', +operand: string | UserId |}
- | {| +operator: 'near' | 'id', +operand: number |} // message ID
- ;
+ | {| +operator: 'near' | 'id', +operand: number |}; // message ID
```

Comment:
nit: small noise formatting change

(The existing formatting here is intentional, for making things slightly easier when adding more branches.)